utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -99,6 +99,10 @@ pub struct StorageParams {
 	/// State cache size.
 	#[clap(long, default_value = "0")]
 	pub state_cache_size: usize,
+
+	/// Include child trees in benchmark.
+	#[clap(long)]
+	pub include_child: bool,
```

Comment:
```suggestion
	pub include_child_trees: bool,
```
Could you also add this to the test in  `bin/node/cli/tests/benchmark_storage_works.rs`, to see that the argument is parsed correctly?

Reply:
changed